test/integration/garbage_collector_test.go

```diff
@@ -43,6 +43,10 @@ import (
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+var trueVar, falseVar = true, false
+var orphanOptions = &api.DeleteOptions{OrphanDependents: &trueVar}
+var nonOrphanOptions = &api.DeleteOptions{OrphanDependents: &falseVar}
```

Comment:
Please write functions to return these things instead of using global variables. If someone accidentally writes to these variables you will have very hard to find bugs.